Makefile

```diff
@@ -36,7 +56,9 @@ all build:
 #   make check
 #   make test
 #   make check WHAT=pkg/build GOFLAGS=-v
-check test:
+check test: export STI_COVER= -cover -covermode=atomic
+check test: export STI_RACE=  -race
+check test: verify
```

Comment:
I still dont' see why you don't just write:  
check test:
  STI_COVER=" - cover -covermode=atomic" STI_RACE=" -race" hack/test-go.sh $(WHAT) $(TESTS)

the example you linked to doesn't "export" the env variable and it's not clear you need to?